portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,145 @@
+                    hospitals. <br><br></li>
+                    <li><b>Navigation to safe locations: </b>
+                    In case of an emergency when a user is walking on the street, pear. 
+                    will provide navigation to the best "safe location" close to you - 
+                    all with one click. "Safe locations" may include restaurants or 
+                    hospitals. In order to find the best safe location, pear. takes 
+                    into account both the proximity to the current location of the 
+                    user and the safety of the trajectory there, using AI to find the 
+                    best possible place and route. The navigation includes a map with 
+                    the highlighted path, as well as an audio giving the directions. <br><br></li>
+                    <li><b>Incognito Mode: </b>
+                    pear. allows you to go to incognito mode while using the app. 
+                    At any time, you can click one button and go to a page that mimics 
+                    a food delivery app, which guarantees the user's safety and privacy. <br><br></li>
+                    <li><b>Angela: </b>
+                    Fake caller app for the user to safely get away from uncomfortable or 
+                    potentially problematic situations. <br></li>
+                </ul>
+            </div>
+
+            <div id="tech">
+            </div>
+
+            <div id="comment">
+                <table style="margin-top:20px;">
+                    <tr>
+                        <td>
+                            <p class="bigHeader">COMMENTS</p>
+                        </td>
+                        <td class="borderCell"></td>
+                    </tr>
+                </table>
+
+                <table>
+                    <tr>
+                        <td style="padding-left:50px;">
+                            <form action="/data" method="POST" target="_top">
+                                <input type="text" name="name" placeholder="How should I call you?" style="width:92%; font-family:'Montserrat', sans-serif; font-weight:300; color:gray; ">
+                                <br><br>
+                                <input type="text" name="comment" placeholder="Write a comment about this project!" style="width:92%; height:70px; font-family:'Montserrat', sans-serif; font-weight:300; color:gray; ">
+                                <br><br>
+                                <input type="submit" value="Submit">
+                            </form>
+                        </td>
+                        <td class="borderCell"></td>
+                    </tr>
+                </table>
+
+                <table style="margin:20px;">
+                    <tr>
+                        <td id="commentCell">
+                        </td>
+                        <td class="borderCell"></td>
+                    </tr>
```

Comment:
As a challenge to you: could you write your comments structure without using "table"?